proto/tabletmanagerdata.proto

```diff
@@ -616,6 +617,7 @@ message ReadVReplicationWorkflowResponse {
     string component_throttled = 14;
   }
   repeated Stream streams = 11;
+  string options = 12;
```

Comment:
If only we had added this instead of `defer_secondary_keys` :(

Reply:
The plan is to move some of the current columns which are essentially options to the json column in one of the upcoming releases and deprecate them.